viadot/flows/customer_gauge_to_adls.py

```diff
@@ -38,6 +38,8 @@ def __init__(
         ] = None,
         start_date: datetime = None,
         end_date: datetime = None,
+        method1_cols: List[str] = None,
+        method2_cols: List[str] = None,
```

Comment:
Is there a way to name these parameters differently to have a clearer view? 
-> example: `unpack_by_field_reference_cols`, `unpack_by_dict_transformer_cols`. 
What do you think?